packages/picasso.js/src/core/chart-components/labels/strategies/bars.js

```diff
@@ -308,13 +306,16 @@ export function placeInBars(
         });
 
         if (label) {
+          const textBounds = approxTextBounds(label, measured, isRotated, bounds, placement.padding);
+          fill = typeof placement.fill === 'function' ? placement.fill(arg, i, textBounds) : placement.fill;
```

Comment:
Don't like adding `textBounds` as a 3rd argument. A better solution IMO would be to extend the first argument with the `textBounds`

Reply:
Fixed!